README.md

```diff
@@ -1,45 +1,75 @@
 # Golden Sample Angular App
 
-This example project is a reference implementation to showcase a number of best practices to use when building a new Angular SPA that leverages Backbase components and libraries. The project is regularly updated, so come back often to check for new improvements.
+This example project is a reference implementation to showcase a number of best practices to use when building a new Angular SPA that leverages Backbase components and libraries. The project is regularly updated, so come back often to check for new improvements. For the best experience, these VSCode extenstions for Nx and Jest should be installed. Jest runner allows you to easly run isolated test as you are developing. Nx Console allows you to easily find and run all the possible Nx Commands.
```

Comment:
Are you using Jest for testing?

Reply:
Jest was setup for Unit Testing as discussed in the previous comments. I confirmed there were no issues in the Angular 13 branch.